### DISCUSSION TITLE
Is the documentation correct?

### DISCUSSION THREAD

**Discussion by shubdragon** [alpinejs/alpine#3008] [4. General]
Link : https://alpinejs.dev/directives/data
![Screenshot_2022-06-27-22-43-29](https://user-images.githubusercontent.com/20332532/175998462-d7fab995-6103-478e-9d7c-187006678403.png)
Should output be baz or bar in second  div?

**Comment by Martleby** [comment]
"bar", I think, since x-text="foo" and in that context foo is set to 'bar'.

Cheers,
Martin.
